presentation/common/src/presentation-common/content/Fields.ts

```diff
@@ -374,13 +379,15 @@ export class NestedContentField extends Field {
     nestedFields: Field[],
     editor?: EditorDescription,
     autoExpand?: boolean,
+    relationshipMeaning?: string,

```

Comment:
This is a breaking change. I'd say let's just leave the public member without ability to set it through constructor.

Reply:
Updated